draft/2022-08-10-this-week-in-rust.md

```diff
@@ -36,9 +36,8 @@ and just ask the editors to select the category.
 ### Project/Tooling Updates
 
 ### Observations/Thoughts
-
+[Basic Operator Overloading with Traits](https://rsdlt.onrender.com/posts/welcome-blog-rust-technology-development-programming-language/)
 ### Rust Walkthroughs
-
```

Comment:
Github is having a tough time resolving the merge conflict...  can this blank line be restored as well? Thank you!